### PR TITLE
Shuttle resprite

### PR DESCRIPTION
**Pull request by PJB3005**
Only affects box because I ran out of ability to care.

### CODE REVIEW

**Comment by ghost** [comment]
looks snazy

**Comment by CodenameB** [comment]
I like the clean, white escape shuttle. A black one implies foreboding or danger.

**Comment by PJB3005** [comment]
@CodenameB that's the point. I want a more dark themed SS13.

**Comment by CodenameB** [comment]
Fair enough, but then you should also change the arrival shuttle and the other stations too, or this just sticks out as weird.

**Comment by Exxion** [comment]
>I want a more dark themed SS13.

But does anyone else?

**Comment by mtpowell** [comment]
the shuttle sprites are pretty shitty and do need an update, dark or not, it looks better

**Comment by AI-Is-Rogue** [comment]
What about a lighter shade of grey? I feel this is good but might be a bit too dark.

**Comment by Exxion** [comment]
They're way too dark.
They look good, but should be reserved for the deathsquad shuttle and the like.

**Comment by LordMyroc** [comment]
The shuttles have needed new sprites for a long while now. This seems a bit too dark for the escape shuttle but I think the other shuttles would look nice with these tiles.

**Comment by PJB3005** [comment]
I fail to see why NT would bias specific shuttles to specific colors. Sure "muh death squad is edgy and evil", but that's dumb.

Why *shouldn't* the shuttle be dark? Is there some hidden lore that says "civil shuttles = white, edgy DS shuttles = black"?

**Comment by yclatious** [comment]
Personally I kinda liked the white and think the dark here is a bit too much but this is some high quality shit so 👍 on me

**Comment by Ait1** [comment]
Why _should_ the shuttle be dark is my question. It looks fine the way it is and it works well with how NT would use a clean white shuttle for civilians just as any other mass corporation would. It gives a manner of professionalism to it's guests/employees. Dark themed just makes death squad and antag ships/shuttles such as raiders or syndicate seem redundant. It'd be no difference in any other shuttle or ship.

**Comment by cadaverouscoyote** [comment]
the lighter grout and dark tiles contrasted by the outline of everything else makes the floor look out of focus, like everything else is floating. in addition, that ugly plating along the wall makes corners like by the L-table look fucked.

also it's stupid and edgy and i hate it

**Comment by PJB3005** [comment]
Yeah but why would raiders want to stand out? "it makes sense duh"

> It looks fine the way it is and it works well with how NT would use a clean white shuttle for civilians just as any other mass corporation would.  It gives a manner of professionalism to it's guests/employees

And... Why would NT use a clean white shuttle? I don't see how white or black gives "professionalism"

**Comment by Ait1** [comment]
For the same reason as I said

> It gives a manner of professionalism to it's guests/employees.

It makes more sense if it were white.

**Comment by cadaverouscoyote** [comment]
> why would raiders want to stand out?

they wouldn't- unlike a rescue vessel, so they'd probably want to blend in with the darkness of space, unlike a rescue vessel

**Comment by Arthurdentist** [comment]
I agree that it ought to be white. Central wouldn't send an escape shuttle like this because it goes against the 'image' they are trying to present. 

Its a company that pretends its main activity is 'research' above anything else. It should always be pushing that 'we are definitely just a research station guys' angle, and that is why the shuttle would be white. Black looks far too 'military'. I don't think this really fits very well at all.

**Comment by PJB3005** [comment]
> CC attempting to present any image.

Nice meme.

If we want a lore contest I could say that central cheaps out on shuttles because they're fucking Nanotrasen and I guarantee you white paint would be too expensive.

It's supposed to look military. It's an *emergency* shuttle. Yes that doesn't apply on our server but if you want to add explicit crew transfer votes and make the shuttle change depending on that then you have an argument.

Being an *emergency* shuttle, it should be "everybody get the fuck in line no time for pretty ness".

**Comment by PJB3005** [comment]
Also this is the kind of shuttle that's used to save the station from plasma fires, meteor storms, and all kind of crap. I'll take military dark over civilian white thank you.

**Comment by Ait1** [comment]
You are absolutely right, it is an _emergency_ shuttle, that's why rescue vessels in almost every navy including any space navy would want the color of rescue to stand out- such as orange or white.

And if we're talking about cost white is always the cheapest option, just look at NASA. Do you see any black vehicles or shuttles? Black is also the most expensive in some cases. Why would you hide rescue?
As @Arthurdentist said they have an image to keep and black would be far too militaristic for a "simple self sustaining research station".

>I'll take military dark over civilian white thank you.

Isn't that the point of a civilian rescue shuttle?

**Comment by PJB3005** [comment]
great so now the argument has devolved into lore.

Why do I even fucking bother jesus christ.

**Comment by Arthurdentist** [comment]
No, image is very important to CC, even if CC is crap at maintaining its image. It puts a lot of effort into it, and even when an emergency occurs it would still try to maintain that image. Do you think that Central Command would admit that Wizards, Changelings, Cultists or The Syndicate all exist? It would never do that. Any 'emergencies' that occur would always be presented as freak fires, electrical disasters and other mistakes. Worth pointing out that white isn't just the colour of research, its also the colour of medical. A medical emergency shuttle would be white. 

You might have a point if the station were explicitly a military station, and that might actually be an interesting idea for a future map. How would a central command 'military' installation differ from a 'research facility'? This is Box station though, which is really the 'base' station and probably the most typical.

**Comment by cadaverouscoyote** [comment]
why _do_ you bother?

**Comment by Arthurdentist** [comment]
Nothing anybody here said was anything less then respectful. You might think that people have been unfair to you in the past but I hardly think that is evidenced here. People were polite and a lot of people actually liked the design. We could have easily reached a compromise.

 Don't act like a goddamned child.

**Comment by PJB3005** [comment]
@cadaverouscoyote because I want to actually improve this entire game being cancerous shit.

Do you ever see TITANIUM WHITE FUCKING SHUTTLES in sci-fi games? no.

**Comment by PJB3005** [comment]
I'm not respriting them. White is fucking ugly. Take the sprites and the script if you want to.

**Comment by ghost** [comment]
white spaceship

**Comment by ghost** [comment]
real life white spaceship

**Comment by Ait1** [comment]
rescue boat

**Comment by ghost** [comment]
subnautica aurora

**Comment by cadaverouscoyote** [comment]
y'know i deleted that because it was just me being pissy and i didn't want to contribute no-quality posts to this discussion but when pjb's around there's really no point

**Comment by ghost** [comment]
Virgin Galactic

**Comment by ghost** [comment]
Star wars spaceship

**Comment by ghost** [comment]
White shuttle is iconic and I really really don't see what's wrong with the current sprites, what's the point of this reskin?

>but I WANT it to be edgy

Oh

File: __DEFINES/lighting.dm [hunk 1]
```diff
@@ -11,7 +11,7 @@
 #define LIGHTING_HEIGHT         1 // height off the ground of light sources on the pseudo-z-axis, you should probably leave this alone
 #define LIGHTING_ROUND_VALUE    1 / 128 //Value used to round lumcounts, values smaller than 1/255 don't matter (if they do, thanks sinking points), greater values will make lighting less precise, but in turn increase performance, VERY SLIGHTLY.
 
-#define LIGHTING_ICON 'icons/effects/lighting_overlay.png' // icon used for lighting shading effects
+#define LIGHTING_ICON 'icons/effects/lighting_overlay.dmi' // icon used for lighting shading effects
```
Comment (Skullyton) [inline __DEFINES/lighting.dm]: why the switch from png to dmi exactly?

**Comment by ghost** [comment]
For some actual criticism on the new sprites.

They're way too dark, everything else sticks out like hell. The tables, the windows, the command airlock, even the secondary floor colors. It just wouldn't wouldn't blend in unless the rest of the shuttle was also reskinned with this extremely dark theme.

The off-grid floors. I get this is the main thing you were going for but man does it just not look good with the grid tile placement of everything else, the chairs look like they're floating, everything not touching a wall looks so wrong when you're used to tile-strict floors.

The walls look like they are lacking detail when not fully lit, they're so dark they blend in with the actual total dark. I mean shit man look at the centcomm walls in the same image.

Also for some reason the floor looks blurry, the hell

**Comment by Blithering** [comment]
Walls are nice, would make a good syndicate shuttle

**Pull request closed without merging**